### PR TITLE
git-config: add German translation

### PR DESCRIPTION
**Pull request by solorisx**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages.de/common/git-config.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# git config
+
+> Verwalten von benutzerdefinierten Optionen für Git Repositories.
+> Diese Optionen können lokal (für das aktiven Repository) or global (für den aktiven Benutzer) sein.
+> Weitere Informationen: <https://git-scm.com/docs/git-config/de>.
+
+- Auflisten von lokalen Konfigurationseinträgen (gespeichert unter `.git/config` im aktiven Repository):
```
Comment (pixelcmtd) [inline pages.de/common/git-config.md:7]:
```suggestion
- Liste nur lokale Konfigurationseinträge (gespeichert unter `.git/config` im aktiven Repository) auf:
```

File: pages.de/common/git-config.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+`git config --list --local`
+
+- Auflisten von global Konfigurationseinträgen (gespeichert unter `~/.gitconfig`):
```
Comment (pixelcmtd) [inline pages.de/common/git-config.md:11]:
```suggestion
- Liste nur globale Konfigurationseinträge (gespeichert unter `~/.gitconfig`) auf:
```

File: pages.de/common/git-config.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+`git config --list --global`
+
+- Auflisten von System-Konfigurationseinträgen (gespeichert unter `/etc/gitconfig`), und deren Speicherort:
```
Comment (pixelcmtd) [inline pages.de/common/git-config.md:15]:
```suggestion
- Liste nur System-Konfigurationseinträge (gespeichert unter `/etc/gitconfig`) und deren Speicherort auf:
```

File: pages.de/common/git-config.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+`git config --list --system --show-origin`
+
+- Ermitteln des Wertes für einen bestimmten Konfigurationseintrag:
```
Comment (pixelcmtd) [inline pages.de/common/git-config.md:19]:
```suggestion
- Gib den Wert eines bestimmten Konfigurationseintrags aus:
```

File: pages.de/common/git-config.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+`git config alias.unstage`
+
+- Setzen des globalen Wertes für einen bestimmten Konfigurationseintrag:
```
Comment (pixelcmtd) [inline pages.de/common/git-config.md:23]:
```suggestion
- Setze den globalen Wert eines bestimmten Konfigurationseintrags:
```

File: pages.de/common/git-config.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+`git config --global alias.unstage "reset HEAD --"`
+
+- Zurücketzen des globalen Wertes für einen bestimmten Konfigurationseintrag auf seinen Standardwert:
```
Comment (pixelcmtd) [inline pages.de/common/git-config.md:27]:
```suggestion
- Setze den globalen Wert eines bestimmten Konfigurationseintrags auf seinen Standardwert zurück:
```

File: pages.de/common/git-config.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+`git config --global --unset alias.unstage`
+
+- Bearbeiten der Git-Konfiguration für das aktive Repository mit dem Standard-Editor:
```
Comment (pixelcmtd) [inline pages.de/common/git-config.md:31]:
```suggestion
- Bearbeite die Git-Konfiguration für das aktuelle Repository mit dem Standard-Editor:
```

File: pages.de/common/git-config.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+`git config --edit`
+
+- Bearbeiten der globalen Git-Konfiguration mit dem Standard-Editor:
```
Comment (pixelcmtd) [inline pages.de/common/git-config.md:35]:
```suggestion
- Bearbeite die globale Git-Konfiguration mit dem Standard-Editor:
```

**Review by pixelcmtd — CHANGES_REQUESTED** [review]
Please read [the Style Guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#imperative-mood) before contributing

File: pages.de/common/git-config.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
+# git config
+
+> Verwalten von benutzerdefinierten Optionen für Git Repositories.
+> Diese Optionen können lokal (für das aktiven Repository) or global (für den aktiven Benutzer) sein.
+> Weitere Informationen: <https://git-scm.com/docs/git-config/de>.
```
Comment (kbdharun) [inline pages.de/common/git-config.md:5]: Thanks for your contribution and welcome to tldr. Is the `de` specifier at last necessary, I think the site would redirect to the correct locale based on your system/browser's language preference.

```suggestion
> Weitere Informationen: <https://git-scm.com/docs/git-config>.
```

Comment (solorisx) [reply pages.de/common/git-config.md:5]: I think it is. For me (german language preference set), it does not redirect to the german site.

Comment (kbdharun) [reply pages.de/common/git-config.md:5]: > I think it is. For me (german language preference set), it does not redirect to the german site.

Thanks for verifying it, I tried now. I can confirm the same.

**Review by kbdharun — APPROVED** [review]

**Review by pixelcmtd — APPROVED** [review]

**Pull request merged**